backend/createStaffAndRooms.py

```diff
@@ -78,7 +78,10 @@ def create_rooms_main(rooms_file, is_hardrock=False, force_roombaht=False):
                         number=elem['Room']
                         )
 
-            features = elem['Room Features (Accessibility, Lakeview, Smoking)'].lower()
+            try:
```

Comment:
Are you still having parsing errors here? I'm not super comfortable with making this bit optional (unless we make it mutable) especially given #66

Reply:
commented on the issue. 
Are we thinking we need these fields to compose the complete hotel name? 
it feels extra to me. would be nice to a have an extensible dataset but ultimately room number is wat the hotel mostly cares about ja?

Reply:
Also if is pref, i can use the new room csv. 
The one i have just contains the rooms types i wanted for 1to1 matching with the short_guest lists i made. i cans change tho!

Reply:
Sheisse. forgot to do something about this. will lewk tmw

Reply:
New room CSV (and any test fixtures we come up with) should use the format listed here. Note some of the parsing semantics will change once #59 lands.